Update dev guide FAQ to introduce the new renew model

r0dneyP3 opened the pull request:
Add section explaining the Valid Until date field.
Update existing FAQ to reflect the new renewal model.

r0dneyP3 committed:
Add section explaining the Valid Until field
Update existing FAQ to reflect the new renewal model.

jpchase commented on developer-guide.md:
```diff
@@ -67,14 +67,24 @@ The [developer console](https://developers.chrome.com/origintrials/#/trials/acti
+      expiration date and cannot be updated, so you must renew to generate a new token in this
+      case. We'll also send you an email to invite you to renew and continue participating in the
+      trial.
+  - Prior to Jan 27, 2021, tokens were issued to be short-lived, lasting 6 weeks until they expire. 
```
Nit: Use the full month name.
```suggestion
  - Prior to January 27, 2021, tokens were issued to be short-lived, lasting 6 weeks until they expire. 
```

jpchase commented on developer-guide.md:
```diff
@@ -169,3 +179,34 @@ The [developer console](https://developers.chrome.com/origintrials/#/trials/acti
       when a valid token is provided. The exclusion percentage varies for each trial, but is
       typically less than 5%.
+  ### 21. What does the _Valid Until_ date mean for my tokens?
+  - Tokens are guaranteed to be accepted by Chrome up to the _Valid Until_ date. It is calculated
```
Nit:
```suggestion
  - Tokens are guaranteed to be accepted by Chrome up to the _Valid Until_ date. This date is calculated
```

jpchase commented on developer-guide.md:
```diff
@@ -169,3 +179,34 @@ The [developer console](https://developers.chrome.com/origintrials/#/trials/acti
+      participating in the trial. 
+  - Before the _Valid Until_ date arrives, we'll send an email to invite you to provide feedback
+      and continue participating in the trial.
+  - Tokens without any feedback in the past 6 weeks will automatically be disabled via remote
```
Nit:
```suggestion
  - Tokens without any feedback in the past 6 weeks will automatically be disabled by a remote
```

jpchase commented on developer-guide.md:
```diff
@@ -169,3 +179,34 @@ The [developer console](https://developers.chrome.com/origintrials/#/trials/acti
+  - Before the _Valid Until_ date arrives, we'll send an email to invite you to provide feedback
+      and continue participating in the trial.
+  - Tokens without any feedback in the past 6 weeks will automatically be disabled via remote
+      processes after a grace period.
```
NIt:
```suggestion
      update process after a grace period.
```

jpchase commented on developer-guide.md:
```diff
@@ -169,3 +179,34 @@ The [developer console](https://developers.chrome.com/origintrials/#/trials/acti
+  - Tokens without any feedback in the past 6 weeks will automatically be disabled via remote
+      processes after a grace period.
+  - We will send a last warning email before disabling the token. This will be your last chance
+      to provide feedback and keep the token active. After that your token will stop working on
```
Maybe use "accepted" as earlier, and be more explicit about what "stop working" means?
```suggestion
      to provide feedback and keep the token active. After that your token will no longer be accepted by
```

jpchase commented on developer-guide.md:
```diff
@@ -169,3 +179,34 @@ The [developer console](https://developers.chrome.com/origintrials/#/trials/acti
+      processes after a grace period.
+  - We will send a last warning email before disabling the token. This will be your last chance
+      to provide feedback and keep the token active. After that your token will stop working on
+      Chrome.
```
... continued:
```suggestion
      Chrome, and will not enable the experimental feature on any page.
```

jpchase commented on developer-guide.md:
```diff
@@ -169,3 +179,34 @@ The [developer console](https://developers.chrome.com/origintrials/#/trials/acti
   - User Subset: A small percentage of Chrome users will always be excluded from the trial, even
       when a valid token is provided. The exclusion percentage varies for each trial, but is
       typically less than 5%.
+  ### 21. What does the _Valid Until_ date mean for my tokens?
```
I suggest defining an explicit anchor, rather than relying on the auto-generated one.
```suggestion
  ### 21. What does the _Valid Until_ date mean for my tokens?  <a name="valid-until"/>
```

jpchase commented on developer-guide.md:
```diff
@@ -169,3 +179,34 @@ The [developer console](https://developers.chrome.com/origintrials/#/trials/acti
+        required to extend the lifespan of the token.
+    - To ensure active trial participants can benefit from the trial without the overhead of
+        repeatedly deploying new tokens.
+### 22. How do I keep my tokens active for the entire trial?
```
Ditto for adding an explicit anchor tag.

r0dneyP3 replied:
```suggestion
### 22. How do I keep my tokens active for the entire trial? <a name="valid-until-feedback"/>
```

jpchase reviewed (commented):
Mostly minor comments

r0dneyP3 committed:
Apply suggestions from code review

Co-authored-by: Jason Chase <chasej@chromium.org>

r0dneyP3 committed:
Update developer-guide.md

Co-authored-by: Jason Chase <chasej@chromium.org>

r0dneyP3 committed:
Add anchor to Q12 for easier reference
Convert self-closing anchor tags to regular <a></a> pair to comply to HTML5 standards

r0dneyP3 commented on developer-guide.md:
```diff
@@ -155,7 +165,7 @@ The [developer console](https://developers.chrome.com/origintrials/#/trials/acti
   care to limit usage to a suitable portion of their traffic.
 
-  ### 20. What are the options for usage retrictions on tokens? <a name="usage-restrictions"/>
+  ### 20. What are the options for usage retrictions on tokens? <a name="usage-restrictions"></a>
```
<a> is not considered void element so we should explicitly write a closing tag, otherwise it's treated with a starting tag without an end, making the next bit of text a link.

jpchase commented on developer-guide.md:
```diff
@@ -67,14 +67,24 @@ The [developer console](https://developers.chrome.com/origintrials/#/trials/acti
+      expiration date and cannot be updated, so you must renew to generate a new token in this
+      case. We'll also send you an email to invite you to renew and continue participating in the
+      trial.
+  - Prior to January 27, 2021, tokens were issued to be short-lived, lasting 6 weeks until they expire. 
```
I think we can change this to February 3.

jpchase reviewed (approved):
Looks good, with one update needed to the date.

r0dneyP3 committed:
Update developer-guide.md
